src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas1OutOfServiceBedEntity.kt

```diff
@@ -15,6 +15,9 @@ import javax.persistence.Table
 
 @Repository
 interface Cas1OutOfServiceBedRepository : JpaRepository<Cas1OutOfServiceBedEntity, UUID> {
+  @Query("SELECT oosb FROM Cas1OutOfServiceBedEntity oosb WHERE oosb.endDate >= CURRENT_DATE ORDER BY oosb.startDate")
```

Comment:
I tend to think that filtering by temporality ought perhaps to come later? And that initially we should return all out of service beds?

Also, is it worth being explicit about the ordering direction e.g `ORDER BY oosb.startDate DESC`.

I for one couldn't say for sure what the default ordering on a date field would be!

Reply:
I've added the `WHERE` clause as it's the default filter described in [APS-847](https://dsdmoj.atlassian.net/browse/APS-847), so no changes to the default behaviour should occur when filtering is fully implemented.

The default sort direction when not specified is `ASC` but I can make this explicit.